docs/source/tutorials/python_api_demo.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### End to End Demo for Cleanlab Studio Python API"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1) Initialize Studio object with your API key"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab_studio import Studio\n",
+    "\n",
+    "api_key = \"<insert_api_key>\"\n",
+    "studio = Studio(api_key)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. Load dataset as pandas dataframe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   index                                        review_text        asin  stars\n",
+      "0      0  This magazine was great for the times but as w...  B00005N7P0    ⭐⭐⭐\n",
+      "1      1  We ordered this magazine for our grandson (the...  B00005N7P0  ⭐⭐⭐⭐⭐\n",
+      "2      2  I didn't receive a full year.  I only receive ...  B00005N7P0     ⭐⭐\n",
+      "3      3  I was hoping for more technical than what was ...  B00005N7P0     ⭐⭐\n",
+      "4      4  I only received one copy of the mag so I could...  B00005N7P0     ⭐⭐\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "df = pd.read_csv(\"https://s.cleanlab.ai/amazon-text-demo.csv\")\n",
+    "print(df.head())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Upload dataset to Studio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
```

Comment:
why is this showing up in stderr? (can see in docs that this is highlighted in red)

is this the case for the actual output? easy fix is to just change this to stdout

Reply:
not sure. might be a tqdm thing. i'll just edit to stdout

Reply:
https://tqdm.github.io/docs/tqdm/#tqdmstd

yeah does seem to be tqdm